src/builtin.jq

```diff
@@ -181,14 +181,8 @@ def combinations(n):
       | combinations;
 # transpose a possibly jagged matrix, quickly;
 # rows are padded with nulls so the result is always rectangular.
-def transpose:
-  if . == [] then []
-  else . as $in
-  | (map(length) | max) as $max
-  | length as $length
-  | reduce range(0; $max) as $j
-      ([]; . + [reduce range(0;$length) as $i ([]; . + [ $in[$i][$j] ] )] )
-  end;
+# Using map(length) turns out to be faster than using max(stream)
```

Comment:
I'm still unsure about the stream version of max to be included or not. Users not following the history will be get confused by this comment; where is `max(stream)`?

Reply:
Let's just drop the sentence, we don't need such performance notes here.

Reply:
OK. Done in https://github.com/jqlang/jq/pull/2758/commits/a20c6d50425251c211bfa5c6608c7049d81bd331